doc/CMakeLists.txt

```
@@ -1,2 +1,122 @@
-add_subdirectory(doxygen)
-add_subdirectory(sphinx)
+# Main documentation targets
+add_custom_target_subproject(doc sequant)
```

```suggestion
add_custom_target_subproject(sequant doc)
```
see https://github.com/ValeevGroup/SeQuant/blob/master/cmake/modules/AddCustomTargetSubproject.cmake#L17